### PR TITLE
Hic opt

### PR DESCRIPTION
**Pull request by yumisims**
## PR checklist

- [ ] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/sanger-tol/treeval/tree/master/.github/CONTRIBUTING.md)
- [ ] Make sure your code lints (`nf-core lint`).
- [ ] Ensure the test suite passes (`nextflow run . -profile test,docker --outdir <OUTDIR>`).
- [ ] Usage Documentation in `docs/usage.md` is updated.
- [ ] Output Documentation in `docs/output.md` is updated.
- [ ] `CHANGELOG.md` is updated.
- [ ] `README.md` is updated (including new tool citations and authors/contributors).

### CODE REVIEW

**Comment by gq1** [comment]
The runner name should be `ubuntu2204-8c`

**Comment by gq1** [comment]
https://github.com/sanger-tol/treeval/blob/d81682fdb1de09eed0f6044dd59cd93889d75490/assets/github_testing/TreeValTinyFullTest.yaml#L34

Does the line have the wrong path?

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -45,6 +45,20 @@ jobs:
...
+          python-version: "3.10"
+
+      - name: Install nf-core
+        run: |
+          pip install nf-core
+
+      - name: Download containers
+        # Forcibly download repo on branch hic_opt and download SINGULARITY containers into the CACHE dir if not found
+        run: |
+          nf-core download sanger-tol/treeval --revision hic_opt --compress none -d --force --outdir sanger-treeval --container-cache-utilisation amend --container-system singularity
```
Comment (gq1) [inline .github/workflows/ci.yml:60]: This step pull all 35 sif image in advance before the run and it seems help lots . 

After downloading, it seems we should run the pipeline differently according to the document. I never tried the nf-core download myself.

https://nf-co.re/tools#downloading-pipelines-for-offline-use

`nextflow run /path/to/download/nf-core-rnaseq-dev/workflow/ --input mydata.csv --outdir results  # usual parameters here`

Comment (DLBPointon) [reply .github/workflows/ci.yml:60]: I can do that.

I've removed the checkout@v4 and how have it running the nf-core downloaded version. Has meant I've had to include another action to get branch names. See more on #234 .

**Pull request merged**